partners/astropy.md

```diff
@@ -0,0 +1,42 @@
+# pyOpenSci's partnership with Astropy
+
+:::{figure-md} astropy-logo
+:class: myclass
+
+<img src="/images/astropy_project_logo.svg" alt="The astropy logo. The logo is an orange egg shape with a white snake spiraling in it. The text of the image says The Astropy Project on three lines." class="mb-1" width="400px">
```

Comment:
Yay for alt-text & image descriptions!

Reply:
Trying hard to ensure we have an accessible guide @namurphy !! i'm so glad you appreciate it!!!